src/app.ts

```
@@ -90,7 +90,7 @@ async function initSitekey() {
 
 			if (initLoaderTitle) initLoaderTitle.innerHTML = `Done!`;
 			if (typeof overlay !== 'undefined' && typeof initLoaderModal !== undefined) {
```

Biome (2.1.2)

[error] Invalid typeof comparison.

Compare with one of the following string literals:


(lint/correctness/useValidTypeof)